python/cudf/cudf/core/series.py

```diff
@@ -157,7 +157,7 @@ def values(self):
 
         return cupy.asarray(
```

Comment:
You need to assert there's no nulls here otherwise you end up with garbage data in the array.